targets/TARGET_STM/TARGET_STM32L0/TARGET_NUCLEO_L073RZ/device/TOOLCHAIN_IAR/stm32l073xx.icf

```diff
@@ -1,7 +1,15 @@
-/* [ROM = 192kb = 0x30000] */
-define symbol __intvec_start__     = 0x08000000;
-define symbol __region_ROM_start__ = 0x08000000;
-define symbol __region_ROM_end__   = 0x0802FFFF;
+#if !defined(MBED_APP_START)
```

Comment:
It seems that this line doesn't compile...

I could see from other targets something like:
if (!isdefinedsymbol(MBED_APP_START)) { define symbol MBED_APP_START = 0x08000000; }

Reply:
Good call. I've updated it.